integration.states.test_network.NetworkTest.test_managed

```
Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/states/test_network.py", line 34, in test_managed
    self.assertEqual('Interface dummy0 is set to be added.', ret[state_key]['comment'])
AssertionError: u'Interface dummy0 is set to be added.' != u'Interface dummy0 is set to be updated:\n'
- Interface dummy0 is set to be added.
?                                ^^  ^
+ Interface dummy0 is set to be updated:

?                               +++ ^  ^ +
```

--- a/salt/modules/rh_ip.py
+++ b/salt/modules/rh_ip.py
@@ -1032,6 +1032,14 @@ def build_interface(iface, iface_type, enabled, **settings):
             rh_major = '7'
         else:
             rh_major = '6'
+    elif __grains__['os'] == 'Amazon':
+        # TODO: Is there a better formula for this? -W. Werner, 2019-05-30
+        # If not, it will need to be updated whenever Amazon releases
+        # Amazon Linux 3
+        if __grains__['osmajorrelease'] == 2:
+            rh_major = '7'
+        else:
+            rh_major = '6'
     else:
         rh_major = __grains__['osrelease'][:1]
 

